server/src/main/scala/com/stratio/crossdata/server/actors/ServerActor.scala

```diff
@@ -61,8 +61,8 @@ object ServerActor {
 }
 
 // TODO it should only accept messages from known sessions
-class ServerActor(cluster: Cluster, sessionProvider: XDSessionProvider)
-  extends Actor with ServerConfig {
+class ServerActor(cluster: Cluster, sessionProvider: XDSessionProvider, serverConfig: ServerConfig)
+  extends Actor {
```

Comment:
This is a silly style In order to minimize changes here and avoid some repetitions, you could import your config attributes:

```scala
import serverConfig._
```